cookbooks/calico/templates/default/compute/bird.conf.erb

```diff
@@ -29,9 +29,10 @@ protocol direct {
 }
 
 # Peer with all neighbours.
-<% @bgp_neighbors.each do |n| %>
-protocol bgp <%= n[:fqdn].gsub(/[^0-9a-z]/i, '') %> {
-  description "Connection to BGP route reflector";
+<% @bgp_neighbors.each_with_index do |n, index| %>
+# <%= n[:fqdn].gsub(/[^0-9a-z]/i, '') %>
```

Comment:
May as well stop the gsub here, given that it's a comment. At least that way we can see what the FQDN really is. Might be helpful for debugging.